### ISSUE TITLE
[Feature Request] Change wallpaper on Login/Startup

### ISSUE THREAD

**Issue by BrodyB** [pop-os/cosmic-epoch#282]
I've only seen this feature on macOS and nowhere else.

# The Issue
When setting the user's wallpaper to a directory of images, the only option Pop!_OS 22.04 (and every other Ubuntu-derivitive) has for when a new wallpaper is rotated in is via a timer. After the timer is up, the wallpaper is changed. But what if I want to set other triggers for a wallpaper to change?

# Request
I'd really like an option in the Wallpapers settings to cycle to a new wallpaper on Login/Startup, so when I turn my computer on, I'm greeted with the "wallpaper of the day," so to speak. Additional settings could be added in, such as On Wake from Suspend. In these cases, the user wouldn't see the change happen, the wallpaper would be chosen and loaded before the desktop environment is displayed.

**Comment by BrodyB** [comment]
This is probably not in the correct repo

**Comment by leviport** [comment]
Indeed, this is a duplicate of https://github.com/pop-os/cosmic-bg/issues/34

We can transfer issues between repos btw. Next time just let us know instead of making a duplicate issue.
